src/gsi/setupq.f90

```diff
@@ -943,7 +1016,7 @@ subroutine setupq(obsLL,odiagLL,lunin,mype,bwork,awork,nele,nobs,is,conv_diagsav
            my_head => null()
         ENDDO
 
-     endif  ! 181,183,187
+     endif  ! i_use_2mq4b block
```

Comment:
@ClaraDraper-NOAA Is this comment correct?  It doesn't seem to be the end of the i_use_2mq4b block.